docs/types/enums.rst

```diff
@@ -52,16 +52,16 @@ the ``Enum.from_enum`` function.
 
 .. code:: python
 
-    graphene.Enum.from_enum(AlreadyExistingPyEnum)
+    Enum.from_enum(AlreadyExistingPyEnum)
```

Comment:
Like mentioned in https://github.com/graphql-python/graphene/pull/992/files#r291840336 each use of class that also exists in standard lib should have import. Best if all examples have all imports required. I personally would keep `import graphene` and `graphene.xxx` as that is a good practice to import packages rather than members.